packages/web-runtime/src/pages/account.vue

```diff
@@ -525,6 +612,100 @@ export default defineComponent({
       })
     })
 
+    const notificationsSettingsFields = computed(() => [
+      { label: $gettext('Event') },
+      { label: $gettext('Event description'), hidden: true },
+      { label: $gettext('In-App'), alignH: 'right' },
+      { label: $gettext('Mail'), alignH: 'right' }
+    ])
+
+    const emailNotificationsOptionsFields = computed(() => [
+      { label: $gettext('Options') },
+      { label: $gettext('Option description'), hidden: true },
+      { label: $gettext('Option value'), hidden: true }
+    ])
+
+    const updateValueInValueList = (value: SettingsValue) => {
+      const index = unref(valuesList).findIndex(
+        (v) => v.identifier.setting === value.identifier.setting
+      )
+
+      if (index < 0) {
+        valuesList.value.push(value)
+        return
+      }
+
+      valuesList.value.splice(index, 1, value)
+    }
+
+    const updateMultiChoiceSettingsValue = async (
+      identifier: string,
+      key: string,
+      value: boolean | string
+    ) => {
+      try {
+        if (typeof value !== 'boolean') {
+          const error = new TypeError(`Unsupported value type ${typeof value}`)
+
+          console.error(error)
+          captureException(error)
+
+          return
+        }
+
+        const currentValue = unref(valuesList).find((v) => v.identifier.setting === identifier)
+
+        const savedValue = await saveValue({
+          identifier,
+          valueOptions: {
+            collectionValue: {
+              values: [
+                ...(currentValue?.value.collectionValue.values.filter((val) => val.key !== key) ||
+                  []),
+                { key, boolValue: value }
+              ]
+            }
+          }
+        })
+
+        updateValueInValueList(savedValue)
+        showMessage({ title: $gettext('Preference saved.') })
+      } catch (error) {
+        captureException(error)
+        console.error(error)
+        showErrorMessage({
+          title: $gettext('Unable to save preference…'),
+          errors: [error]
+        })
+      }
+    }
+
+    const updateSingleChoiceValue = async (
+      identifier: string,
+      value: { displayValue: string; value: { stringValue: string } }
+    ): Promise<void> => {
+      try {
+        const savedValue = await saveValue({
+          identifier,
+          valueOptions: { stringValue: value.value.stringValue }
+        })
+
+        updateValueInValueList(savedValue)
+        showMessage({ title: $gettext('Preference saved.') })
+      } catch (error) {
+        captureException(error)
+        console.error(error)
+        showErrorMessage({
+          title: $gettext('Unable to save preference…'),
+          errors: [error]
+        })
+      }
+    }
+
+    const canConfigureSpecificNotifications = computed(
+      () => capabilityStore.capabilities.notifications.configurable
```

Comment:
this needs implementation in the server side, right?

Reply:
Correct. For now the whole notifications settings lives in a PR https://github.com/owncloud/ocis/pull/10706

Reply:
ahh, great.